### DISCUSSION TITLE
Compile-time codeblock execution

### DISCUSSION THREAD

**Discussion by FraserLee** [iamcco/markdown-preview.nvim#377] [Ideas]
Hiya. I'm planning on adding jupyter-notebook style python codeblock execution on compile time (inserting stdout as a secondary codeblock below one flagged to be executed), as it would be incredibly helpful for my usecase. Is this close enough to the project goals, or should I just fork it instead?

**Comment by FraserLee** [comment]
I've now created [a tool](https://github.com/FraserLee/execute.md) to do this. Will work on integrating it in a sec

**Reply by blank-manash** [reply]
Can't you do it directly from vim itself?
